### PR TITLE
We don't allow to force the https anymore. Removed unnecesary specs

### PR DESCRIPTION
**Pull request by javierarce**
This PR removes the option to force the `https` protocol of a static map and fixes the specs accordingly (as in removing them completely)

### CODE REVIEW

**Comment by javierarce** [comment]
@javisantana: can you have a look, please?

File: src/vis/image.js [hunk 1]
```diff
@@ -239,7 +239,7 @@
       this.options.tiler_protocol = protocol;
       this.options.tiler_port     = port;
 
-      if (this.userOptions.https || this.imageOptions.vizjson.indexOf("https") === 0) {
```
Comment (javisantana) [inline src/vis/image.js]: this is not needed since if the viz.json is requested by https the url it uses for the tiler is https (since 3.14 version)

**Comment by javisantana** [comment]
:es:

File: src/vis/image.js (hunk 1 above)
Comment (javierarce) [inline src/vis/image.js]: Ok, I'll remove it. Thanks!

**Pull request merged**